src/coreclr/vm/comsynchronizable.cpp

```diff
@@ -321,6 +305,17 @@ void ThreadNative::Start(Thread* pNewThread, int threadStackSize, int priority,
     }
 }
 
+extern "C" void QCALLTYPE ThreadNative_Start(QCall::ThreadHandle thread, int threadStackSize, int priority, BOOL isThreadPool, PCWSTR pThreadName)
+{
+    QCALL_CONTRACT;
+
+    BEGIN_QCALL;
+
+    StartThread(thread, threadStackSize, priority, isThreadPool, pThreadName);
```

Comment:
Does `StartThread` need to be a separate method? I think the logic can be inline here.